### DISCUSSION TITLE
KoPackageMatchingPathProviderCore.hasMatchingPath fails on windows

### DISCUSSION THREAD

**Discussion by Hoagiex** [LemonAppDev/konsist#864] [Report a Problem]
### Tell us what version of Konsist you use.

0.13.0

### Was this something which used to work for you, and then stopped?

0.13.0 is the first version we used.

### Describe the problem

hasMatchingPath fails on windows, due to having a different path seperator

`    override val hasMatchingPath: Boolean
        get() =
            path
                .replace("/", ".")
                .endsWith(fullyQualifiedName + "." + containingFile.nameWithExtension)`

### Actual behaviour

Test succeeds on (e.g. Macos), but fails on windows

### Expected behaviour

Succeeds on windows

**Comment by Hoagiex** [comment]
`    override val hasMatchingPath: Boolean
        get() =
            path
                .replace(File.separator, ".")
                .endsWith(fullyQualifiedName + "." + containingFile.nameWithExtension)`

**Comment by igorwojda** [comment]
Thx for report. Will be fixed in next release (coming in few days).
https://lemonappdev.atlassian.net/browse/KON-593

**Comment by nataliapeterwas** [comment]
A new version v0.14.0 has been released. It contains the fix.
See [v0.14.0 release](https://github.com/LemonAppDev/konsist/releases/tag/v0.14.0)
